src/main/java/home/DirectionsController.java

```diff
@@ -509,7 +514,8 @@ public void setUnits() {
      */
     public void sendMapToPhone(){
         TextingService textSender = new TextingService();
-        textSender.textMap("+19787298044",printDirections(makeDirections(path)));
+        //this grabs the employee ID from the Application state and uses that to get the employee from the database, whose phone we want to use. and sends them the directions.
+        textSender.textMap(getDatabaseService().getEmployee(getApplicationState().getEmployeeID()).getPhone(),printDirections(makeDirections(path)));
```

Comment:
This is somewhat inefficient - getting the employee from the db each time. Why don't you put the entire logged in employee object in the application state?